### DISCUSSION TITLE
Ideal solution for Pizza-Delivery?

### DISCUSSION THREAD

**Discussion by PizzaProgram** [opengisch/QField#2985] [Q&A]
Sorry, if my question is too obvious for professionals like You, but we are looking for a way to use an opensource map system for pizza-delivery on our webpages.
_But all these infos, plugins, blogs and codes are just overwhelming... so I'm here to ask for help to clarify._

## Am I looking at the right place here?  

We are serving 30 restaurant web-pages with the same (self made PHP / JS) engine.
We would like to:
 - Set up delivery areas (poligons) to show where the current pizzeria is delivering.
 - Allow the pizzeria owners to edit those areas easily (by disabling far away ones if too busy + change prices).
 - Show those coloured polygons to the users over street names + **satellite images** 
 - and allow for them to place a marker where they want to order to be delivered.
 - Show the pizza-boy (car) position on the status page so the customer can see if it's nearing.

All this would be limited to 1 country only. 

(PS: Originally I've found [this answer](https://gis.stackexchange.com/a/376995) and came to this site.)

**Comment by suricactus** [comment]
Hi @PizzaProgram . It is possible to use the open-source stack to achieve most of your goals. Please contact info at opengis.ch for details.
